File: src/cpp/1431_kids_greatest_number.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
    vector<bool> greatestNumber;
    int largest = 0;   
    for(int i = 0;i < candies.size(); i++)
    {
           largest = max(largest, candies[i]);
    }
    for (int i = 0; i < candies.size(); i++){
//      int current = candies[i] + extraCandies; // you can use extra variable
        if (candies[i] + extraCandies < largest) {
            greatestNumber.push_back(false);
        }
        else {
            greatestNumber.push_back(true);
        }
        }
    return greatestNumber;
    }
};

File: src/python/1431_kids_greatest_number.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        greatestNumber = []
        for i in range(len(candies)):
            if candies[i] + extraCandies >= max(candies):
                greatestNumber.append(True)
            else:
                greatestNumber.append(False)
        return greatestNumber